Added descriptions about slow node detection (#6174)
pingcap/docs: tikv-configuration-file.md

+ Default value: `1`
+ Minimum value: greater than `0`

### `inspect-interval`

> + At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 

Reviewer: ```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore thread. This parameter specifies the interval of the inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 
```

> + At a certain interval, TiKV inspects the latency status of the Raftstore thread. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 

Reviewer 2: ```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 
```

> + Judges whether the TiKV node is slow based on the ratio of inspected latency. 

Reviewer 2: ```suggestion
+ Judges whether the TiKV node is slow based on the ratio of timeout inspection. 
```

> + Default value: 500ms
> + Minimum value: 1ms

Reviewer: ```suggestion
+ Default value: `"500ms"`
+ Minimum value: `"1ms"`
```

+ Controls whether to enable batch processing of the requests. When it is enabled, the write performance is significantly improved. 
+ Default value: `true`

### `inspect-interval`

> + At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of the inspection. If the latency exceeds this value, that inspection is marked as timeout. 

Reviewer: ```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of the inspection. If the latency exceeds this value, this inspection is marked as timeout. 
```

> + At a certain interval, TiKV inspects the latency of the Raftstore thread. This parameter specifies the interval of the inspection. If the latency exceeds this value, the Raftstore thread is marked as timeout. 

Reviewer 2: ```suggestion
+ At a certain interval, TiKV inspects the latency of the Raftstore component. This parameter specifies the interval of the inspection. If the latency exceeds this value, that inspection is marked as timeout. 
```

Reviewer: 中文里用的是 Raftstore 线程

Reviewer 2: 正确的描述还是还是 inspection，感觉中文的需要改一下，merge comment 的时候没有注意到这个地方

Reviewer: 中文已在 https://github.com/pingcap/docs-cn/pull/6908 中修改。PTAL
